src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64XFormsDotNet.apkdesc

```diff
@@ -5,226 +5,226 @@
       "Size": 3568
     },
     "assemblies/FormsViewGroup.dll": {
-      "Size": 7236
+      "Size": 7251
     },
     "assemblies/Java.Interop.dll": {
-      "Size": 65412
+      "Size": 65415
     },
     "assemblies/Microsoft.Win32.Primitives.dll": {
-      "Size": 4860
+      "Size": 4826
     },
     "assemblies/Mono.Android.dll": {
-      "Size": 454745
+      "Size": 476882
```

Comment:
I think this is correct? Now that `AndroidEnvironment.GetHttpMessageHandler()` preserves `AndroidMessageHandler`, the various `Java.Net.*` APIs are also preserved in `Mono.Android.dll`?

Reply:
I think it's allright